cmd/kubevpn/cmds/dev.go

```diff
@@ -87,16 +87,18 @@ Startup your kubernetes workloads in local Docker container with same volume、e
 			}
 			err = cmd.Flags().Parse(args[1:])
 			if err != nil {
+				err = errors.Wrap(err, "cmd.Flags().Parse(args[1:]): ")
```

Comment:
how about "cmd.Flags().Parse(args[1:]): " --> fmt.Sprinf("failed to parse args: %v", args[:1]), this should more clear ?